library/CHART-TEMPLATE/templates/config-map-of-examples.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "common.naming.fullname" . }}-examples
+  namespace: {{ .Release.Namespace }}
+data:
+  lowDataMode: Low data mode {{ if include "common.lowDataMode" . }}enabled{{ else }}disabled{{ end }}
```

Why having something this complex instead of just
```suggestion
  lowDataMode: {{ include "common.lowDataMode" . | quote }}
```

---

Solved in [9b9b76e](https://github.com/newrelic/helm-charts/commit/9b9b76e3ccfbf000303049efe883be4483698ad4)